webmessaging/with-options/one-arg.html

```diff
@@ -4,11 +4,10 @@
 <script src="/resources/testharnessreport.js"></script>
 <div id="log"></div>
 <script>
-test(function() {
+async_test(t => {
   postMessage('');
-  onmessage = this.step_func(function(e) {
-    assert_equals(e.data, '');
-    this.done();
+  onmessage = t.step_func_done(e => {
+  	assert_equals(e.data, '');
   });
-});
+ });
```

Comment:
This indentation seems to be in error unless I'm missing something.
```suggestion
});
```